unity-client/Assets/Scripts/MainScripts/Debugging/WSS/WSSController.cs

```diff
@@ -279,26 +278,32 @@ void Update()
                                 DCL.Environment.i.debugController.SetEngineDebugPanel();
                                 break;
                             case "SendSceneMessage":
-                                sceneController.SendSceneMessage(msg.payload);
+                                SceneController.i.SendSceneMessage(msg.payload);
                                 break;
                             case "LoadParcelScenes":
-                                sceneController.LoadParcelScenes(msg.payload);
+                                SceneController.i.LoadParcelScenes(msg.payload);
                                 break;
                             case "UnloadScene":
-                                sceneController.UnloadScene(msg.payload);
+                                SceneController.i.UnloadScene(msg.payload);
+                                break;
+                            case "Reset":
+                                SceneController.i.UnloadAllScenesQueued();
+                                break;
+                            case "CreateUIScene":
+                                SceneController.i.CreateUIScene(msg.payload);
+                                break;
+                            case "BuilderReady":
+                                Main.i.BuilderReady();
+                                break;
+                            case "UpdateParcelScenes":
+                                SceneController.i.UpdateParcelScenes(msg.payload);
```

Comment:
all of the `SceneController.i` used here should use `Environment.i.sceneController` right?

Reply:
Yep, something went wrong in a merge, ty for noticing. Fixed.